src/app/modules/familymigration/familymigrationFeedback.js

```diff
@@ -97,13 +97,16 @@ fm.directive('fmFeedback', ['IOService', function (IOService) {
         var setFeedbackVisibility = function (didMatch) {
           $scope.conf.reasonForNotMatch.hidden = true
           $scope.conf.matchOther.hidden = true
-          if (didMatch === 'no' && $scope.passed) {
-            $scope.conf.reasonForNotMatch.hidden = false
+
+          if (didMatch) {
```

Comment:
didMatch implies a boolean result but it looks like there are three states here - did match, didn't match and did match is undefined?  Feels wrong but I'm not sure what the 3rd case means.

Reply:
I agree with this - I'll have a quick think and update.

Undefined is for the initial setting of visibility, where no selection has been made.